cleanlab_studio/studio/studio.py

```diff
@@ -231,6 +231,10 @@ def delete_project(self, project_id: str) -> None:
         api.delete_project(self._api_key, project_id)
         print(f"Successfully deleted project: {project_id}")
 
+    def get_model(self, model_id: str) -> inference.Model:
+        """Creates model object from model ID, to use for inference."""
```

Comment:
Can you merge the latest `main` into this branch and update docstrings to match the format used there (there's a link to the docstring style guide in the README)? Also make sure to document all args and the return value for user facing functions